osutil/exec.go

```diff
@@ -162,6 +162,26 @@ func KillProcessGroup(cmd *exec.Cmd) error {
 	return syscallKill(-pgid, syscall.SIGKILL)
 }
 
+// truncateOutput truncates input data by maxLines and maxBytes (whatever is reached first).
+func truncateOutput(data []byte, maxLines, maxBytes int) bytes.Buffer {
```

Comment:
I was under the impression that we want to keep the last `maxLines` lines that are in total no more than `maxBytes`.